exercises/ruby-1-b/.meta/config.json

```diff
@@ -3,39 +3,72 @@
   "test_file": "ruby_1_b_test.rb",
   "tests": [
     {
-      "test": "test_one_word_with_one_vowel",
```

Comment:
The `"test"` property should contain the name as it is found in the test runner, which is usually the test name. In this case, `"test"` should thus equal `"test_find_by_empty_isbn_returns_nil"`. The `"name"` property is required and should contain a human-readable version of the test. In this case, probably something like `"find by empty isbn returns nil"`.

Could you update the other entries too?